subprojects/docs/src/docs/userguide/authoring-builds/build_lifecycle.adoc

```diff
@@ -57,76 +72,82 @@ include::{snippetsPath}/buildlifecycle/basic/tests/buildlifecycle.out[]
-
-
 [[sec:initialization]]
 == Initialization
 
-How does Gradle know whether to do a single or multi-project build?
-If you trigger a multi-project build from a directory with a `settings.gradle` file, Gradle uses it to configure the build.
-Gradle also allows you to execute the build from within any subproject taking part in the build.footnote:[Gradle supports partial multi-project builds (see <<intro_multi_project_builds.adoc#intro_multi_project_builds,Executing Multi-Project Builds>>).]
-If you execute Gradle from within a project with no `settings.gradle` file, Gradle looks for a `settings.gradle` file in the following way:
+In the initialization phase, Gradle detects the set of projects and included builds participating in the build.
+Then, Gradle evaluates the settings file. Finally, Gradle instantiates `Project` instances for each project.
 
-* It looks for `settings.gradle` in parent directories.
-* If not found, the build is executed as a single project build.
-* If a `settings.gradle` file is found, Gradle checks if the current project is part of the multi-project hierarchy defined in the found `settings.gradle` file.
-If not, the build is executed as a single project build. Otherwise a multi-project build is executed.
+=== Detect Settings File
 
-What is the purpose of this behavior? Gradle needs to determine whether the project you are in is a subproject of a multi-project build or not.
-Of course, if it is a subproject, only the subproject and its dependent projects are built, but Gradle needs to create the build configuration for the whole multi-project build (see <<multi_project_configuration_and_execution.adoc#configuration_and_execution,Configuration and Execution>>).
-If the current project contains a `settings.gradle` file, the build is always executed as:
+When you run Gradle in a directory that contains a `settings.gradle` file, Gradle uses that `settings.gradle` file to initialize the build.
+You can run Gradle within any subproject of the build.footnote:[Gradle supports partial multi-project builds (see <<intro_multi_project_builds.adoc#intro_multi_project_builds,Executing Multi-Project Builds>>).]
+When you run Gradle in a directory that contains *no* `settings.gradle` file:
 
-* a single project build, if the `settings.gradle` file does not define a multi-project hierarchy
-* a multi-project build, if the `settings.gradle` file does define a multi-project hierarchy.
+. Gradle looks for a `settings.gradle` file in parent directories.
+. If Gradle finds a `settings.gradle` file, Gradle checks if the current project is part of the multi-project build. If so, Gradle builds as a multi-project.
+. If Gradle does not find a `settings.gradle` file, Gradle builds as a single project.
```

Comment:
I would prefer to avoid explicitly using only the Groovy DSL or only the Kotlin DSL name for the settings file, and either generically call it "settings file" or say something like `settings.gradle(.kts)`.

Reply:
Agreed. Updated usages to either read "settings file" or include the (.kts). Thanks for pointing this out!